tests/purs/failing/MultipleErrors2.purs

```diff
@@ -1,5 +1,4 @@
--- @shouldFailWith UnknownName
--- @shouldFailWith UnknownName
```

Comment:
Previously, multiple `UnknownName` errors would be reported if there were multiple unknown names, but now only the first one will be reported.  If that behavior is OK, then I should delete this test.

Reply:
I think that's an unfortunate regression we should think about. It's tricky because the type checker does abort with a single error and doesn't let you give out multiple ones. Also with the much larger size of the new error messages, reporting multiple of them might end up flooding the terminal too much.

Reply:
Yeah I'd much prefer to keep the behaviour of reporting multiple errors if we possibly can.

Reply:
I was able to fix this! See https://github.com/purescript/purescript/pull/3619#pullrequestreview-233321885